### PR TITLE
[exclusive-accordion] Discuss accordions that have exactly one item open

### PR DESCRIPTION
**Pull request by dbaron**
Addresses part of #752 and some related issues.

### CODE REVIEW

**Comment by dbaron** [comment]
Preview: [Research](https://deploy-preview-753--open-ui.netlify.app/components/accordion.research/), [Explainer](https://deploy-preview-753--open-ui.netlify.app/components/accordion.explainer/)

File: site/src/pages/components/accordion.explainer.mdx [hunk 1]
```diff
@@ -234,6 +235,18 @@ See https://github.com/whatwg/html/issues/8864 .
...
 of markup that is semantically pairs of headings and sections.)
 
+### Accordions with exactly one item open
+
+Some accordions (such as the one on the Pixel product page above)
+always have exactly one item open,
+and do not allow zero items to be open.
+(This could be called an exact-exclusive accordion.)
+This is a somewhat more complicated widget,
+because the open/close control diappears from the item that is open.
```
Comment (mfreed7) [inline site/src/pages/components/accordion.explainer.mdx:245]: In my look at that page, the "down arrow" widget is still there, it just doesn't do anything. This text is fine, just pointing it out.

File: site/src/pages/components/accordion.explainer.mdx [hunk 2]
```diff
@@ -234,6 +235,18 @@ See https://github.com/whatwg/html/issues/8864 .
...
+It's not clear how such widgets could be modeled with this design.
```
Comment (mfreed7) [inline site/src/pages/components/accordion.explainer.mdx:246]: ...other than via Javascript?

**Review by mfreed7 — APPROVED** [review]
LGTM!

File: site/src/pages/components/accordion.explainer.mdx (hunk 1 above)
Comment (dbaron) [reply site/src/pages/components/accordion.explainer.mdx]: On the pixel product page, it disappears... but on the FAST design system demo page, it indeed becomes disabled.  I'll revise this to say "or becomes disabled".

File: site/src/pages/components/accordion.explainer.mdx (hunk 2 above)
Comment (dbaron) [reply site/src/pages/components/accordion.explainer.mdx]: yeah, mentioned that

**Pull request merged**